tests/packagedcode/data/cocoapods/podfle.lock/braintree_ios_Podfile.lock.expected.json

```diff
@@ -0,0 +1,662 @@
+[
+  {
+    "type": "pods",
+    "namespace": null,
+    "name": "Expecta",
+    "version": "1.0.6",
+    "qualifiers": {},
+    "subpath": null,
+    "primary_language": "Objective-C",
+    "description": null,
+    "release_date": null,
+    "parties": [],
+    "keywords": [],
+    "homepage_url": null,
+    "download_url": null,
+    "size": null,
+    "sha1": null,
+    "md5": null,
+    "sha256": null,
+    "sha512": null,
+    "bug_tracking_url": null,
+    "code_view_url": null,
+    "vcs_url": null,
+    "copyright": null,
+    "license_expression": null,
+    "declared_license": null,
+    "notice_text": null,
+    "root_path": null,
+    "dependencies": [],
+    "contains_source_code": null,
+    "source_packages": [],
+    "extra_data": {},
+    "purl": "pkg:pods/Expecta@1.0.6",
+    "repository_homepage_url": "https://cocoapods.org/pods/Expecta",
+    "repository_download_url": null,
+    "api_data_url": "https://github.com/CocoaPods/Specs/blob/master/Specs/e/9/5/Expecta/1.0.6/Expecta.podspec.json"
```

Comment:
Use the raw, direct download URL such as in https://raw.githubusercontent.com/CocoaPods/Specs/master/Specs/e/9/5/Expecta/1.0.6/Expecta.podspec.json